packages/opentelemetry-exporter-collector/src/CollectorExporter.ts

```diff
@@ -44,7 +44,7 @@ export class CollectorExporter implements SpanExporter {
   readonly logger: Logger;
   readonly hostName: string | undefined;
   readonly attributes?: Attributes;
-  private _isShutdown: boolean = false;
+  private _isShutdown = false;
```

Comment:
Can we remove this rule? Generally, I don't see a problem with setting explicit types in these cases.

Reply:
It refers to https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md

Reply:
Yes I know the rule. I was asking if we can disable it or set `ignoreProperties: true`